### ISSUE TITLE
The miners' stats are broken

### ISSUE THREAD

**Issue by BTKIROS** [UnamSanctam/SilentCryptoMiner#2178]
I was on reg.ru hosting, there for some reason the configuration doesn't work, it doesn't change. But I fixed it successfully on another hosting - 000webhostapp.com
But there is something wrong with the speed, on the old hosting shows that 12 miners online, and on the new one with the same base, shows that none of them are working, although yesterday everything was fine, what is the problem? First 1 miner was online, now 0. But it shows on the old hosting that there are 12 online, what is wrong? Can it be that the two hosting somehow conflict with each other? Right now your panel is running on two hosting sites with the same databases.

Miner Statistics (online). On the new hosting abruptly all shut down:
![изображение](https://github.com/UnamSanctam/SilentCryptoMiner/assets/160104683/65887caf-4653-44db-9641-8e1eeb842a03)

Same stats, but that's on the old hosting:
![изображение](https://github.com/UnamSanctam/SilentCryptoMiner/assets/160104683/3c9eda7d-c545-4dae-be7b-e39eab7d8338)

Hashrate on new hosting (dropped as well as miners):
![изображение](https://github.com/UnamSanctam/SilentCryptoMiner/assets/160104683/205cee47-73be-4243-87d3-2b8a95e7fde2)

Same hashrate stats, but on the old hosting:
![изображение](https://github.com/UnamSanctam/SilentCryptoMiner/assets/160104683/33cb7950-b8bc-4fdb-a798-f9e73e3cdecd)

**Comment by UnamSanctam** [comment]
Are the miners configured to connect to your new hosting? And have you made sure that the database folder and file are writable?

**Comment by BTKIROS** [comment]
Yes, the miners are connected to the hosting. Yesterday everything was working fine, stats were shown, but by the evening for some reason everything shut down. I also gave all permissions to the files on the hosting.

**Comment by UnamSanctam** [comment]
It's possible that 000webhost as blocked it, since they are known for doing that. Are the miners online in the "Miners" tab?

**Comment by BTKIROS** [comment]
In the 000webhost based panel, only 1 of 142 is online. 139 offline and 1 pending

**Comment by UnamSanctam** [comment]
There are four main possibilities, 1) web panel files weren't fully extracted, 2) database isn't writable (though since 1 miner is online it should be), 3) web panel can't be contacted or 4) the miners aren't configured to connect to the web panel.

**Comment by BTKIROS** [comment]
Got it. I'll try to do something. Thank u anyway :)

**Comment by BTKIROS** [comment]
Unam, apparently the new hosting doesn't want to register miners from the old hosting (new miners seem to register without problems). To move the miners from the old hosting, I just moved all the Web Panel files to the new host. On the first day it registered them and showed their statistics, but now the miners of the old hosting are displayed as "offline", although if you go to the old hosting, they are actually "online" and they are working. 

Actually, such a question: is it possible to transfer miners from the old hosting to the new hosting so that they are displayed without problems?

> There are four main possibilities, 1) web panel files weren't fully extracted, 2) database isn't writable (though since 1 miner is online it should be), 3) web panel can't be contacted or 4) the miners aren't configured to connect to the web panel.

**Comment by UnamSanctam** [comment]
Are the URLs different? If so did you change the API Endpoint URL of the old miners to the new URL? If you have a "Remote Configuration" then you can use the "api-endpoint" option to set it.
